connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -45,6 +52,9 @@
     private static String sPaymentCancelUri;
     private static String sPaymentSuccessUri;
     private static SdkProfile sdkProfile;
+    private static ConnectivityManager connectivityManager;
+    private static Network cellularNetwork;
```

Comment:
Yes, good question.
I haven't managed to test that yet on a real device, as I've only tested on the emulator.
I'm waiting for the final patch to snowball, and then I'll do more thorough tests on a real device and will test what you suggested here.